src/sentry/apidocs/hooks.py

```diff
@@ -87,7 +87,7 @@ def __write_ownership_data(ownership_data: dict[ApiOwner, dict]):
         index += __get_line_count_for_team_stats(ownership_data[team])
     dir = os.path.dirname(os.path.realpath(__file__))
     file_to_write = open(f"{dir}/{_OWNERSHIP_FILE}", "w")
-    file_to_write.writelines(json.dumps(processed_data, indent=4))
+    file_to_write.writelines(json.dumps_experimental("apidocs.enable-orjson", processed_data))
```

Comment:
Is there a test that depend on indent=4?

Reply:
AFAICT all these kwargs are actually ignored, but maybe I'm misunderstanding.

Reply:
Oh shoot, this is builtin `json`, not `sentry.utils.json`. I'm reverting the `apidocs` part for now.